core/document/src/color.rs

```diff
@@ -1,5 +1,10 @@
 use serde::{Deserialize, Serialize};
 
+/// Structure that represent a color\
+/// internally alpha is stored as f32 that go from 0.0 (transparent) to 1.0
+/// the other components (rgb) are stored as f32 that go from 0.0 up to f32::MAX,
```

Comment:
Capitalize the start of the new sentence (I'll stop mentioning this now, but please end each sentence with a period and start the next one with an uppercase letter, unless the entire comment is only one sentence then it doesn't need a period but should still start capitalized).

Uppercase "rgb" -> "RGB". Probably also a good idea to put all the `f32`, `0.0`, `f32::MAX`, etc. values in backticks to make those inline code.